src/extension/commands/commandRegistry.ts

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+'use strict';
+
+import { inject, injectable } from 'inversify';
+import { IWorkspaceService } from '../../client/common/application/types';
+
+import { IDisposable } from '../../client/common/types';
+import { JupyterCommandLineSelectorCommand } from './commandLineSelector';
+import { JupyterServerSelectorCommand } from './serverSelector';
+
+@injectable()
+export class CommandRegistry implements IDisposable {
+    private readonly disposables: IDisposable[] = [];
+    constructor(
+        @inject(JupyterServerSelectorCommand) private readonly serverSelectedCommand: JupyterServerSelectorCommand,
```

Comment:
These two commands were under 'interactive' before. I moved them to a separate location.